frontend/pages/SoftwarePage/SoftwareAddPage/SoftwareCustomPackage/helpers.tsx

```diff
@@ -31,6 +31,29 @@ export const getErrorMessage = (err: unknown) => {
     );
   } else if (reason.includes("Secret variable")) {
     return reason.replace("missing from database", "doesn't exist");
+  } else if (reason.includes("Unable to extract necessary metadata")) {
+    return (
+      <>
+        Couldn&apos;t add. Unable to extract necessary metadata.{" "}
+        <CustomLink
+          url={`${LEARN_MORE_ABOUT_BASE_LINK}/package-metadata-extraction`}
+          text="Learn more"
+          newTab
+        />
+      </>
+    );
+  } else if (reason.includes("Fleet couldn't read the version from")) {
```

Comment:
@ghernandez345 I added this back but wasn't sure, lmk https://github.com/fleetdm/fleet/pull/25008/files#r1898638620

Reply:
looks good thanks for catching